devel/python/CMakeLists.txt

```diff
@@ -1,28 +1,40 @@
 include(cmake/find_python_module.cmake)
-include(cmake/python_module_versions.cmake) # finds version
+include(cmake/pycmake.cmake) # finds version
```

Comment:
To facilitate simpler use of the `pycmake` as an external module we could:

1. Put it somewhere "funny" - e.g. `cmake/Modules/pycmake.cmake`.
2. Moidfy the `cmake` search path:

```txt
list( APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/Modules")
```
3. Use an `include( pycmake )` - without prefix path and without extension.